mllib/src/main/scala/org/apache/spark/ml/classification/OneVsRest.scala

```diff
@@ -110,9 +110,9 @@ final class OneVsRestModel private[ml] (
           (predictions: Map[Int, Double], prediction: Vector) => {
             predictions + ((index, prediction(1)))
           }
-        val updateUdf = callUDF(update, mapType, col(accColName), col(rawPredictionCol))
+        val updateUDF = callUDF(update, mapType, col(accColName), col(rawPredictionCol))
```

Comment:
Try this:

~~~scala
        val update = udf { (predictions: Map[Int, Double], prediction: Vector) =>
          predictions + ((index, prediction(1)))
        }
        val transformedDataset = model.transform(df).select(columns : _*)
        val updatedDataset = transformedDataset.withColumn(
          tmpColName, update(col(accColName), col(rawPredictionCol)))
~~~